n26/core/templates/cotton/n26/flair_link.html

```
@@ -27,20 +27,27 @@
-silently dropped, and an unsized svg is 300×150, which wraps under a short
+silently dropped, and an unsized svg is 300x150, which wraps under a short
 word. whitespace-nowrap holds the text and the badge as one unit so a 1em
-mark cannot take the next line by itself. slot.strip because an artwork
-file's own leading and trailing newlines would otherwise pad the badge.
-{% endcomment %}<c-n26.link href="{{ href }}" underline="{{ underline }}" tone="{{ tone }}" class="inline-flex items-center whitespace-nowrap {{ class }}">{{ text }}<c-slot name="trailing">{% if slot.strip %}<span class="ml-[0.25em] n26-icon-inline shrink-0 [--n26-icon-size:1em]" {% if label %}role="img" aria-label="{{ label }}" title="{{ label }}"{% else %}aria-hidden="true"{% endif %}>{{ slot.strip|safe }}</span>{% endif %}</c-slot></c-n26.link>
\ No newline at end of file
+mark cannot take the next line by itself. wrap drops that hold so a sentence
+can break, and skips .n26-icon-inline: that class's vertical-align is for a
+1em mark, and a taller pill rides up into the line above. slot.strip because
+an artwork file's own leading and trailing newlines would otherwise pad the badge.
+{% endcomment %}<c-n26.link href="{{ href }}" underline="{{ underline }}" tone="{{ tone }}" class="{% if wrap %}inline{% else %}inline-flex items-center whitespace-nowrap{% endif %} {{ class }}">{{ text }}<c-slot name="trailing">{% if slot.strip %}<span class="{% if wrap %}ml-1.5 inline-block align-middle{% else %}ml-[0.25em] n26-icon-inline shrink-0 [--n26-icon-size:1em]{% endif %}" {% if wrap %}{% elif label %}role="img" aria-label="{{ label }}" title="{{ label }}"{% else %}aria-hidden="true"{% endif %}>{{ slot.strip|safe }}</span>{% endif %}</c-slot></c-n26.link>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)